### DISCUSSION TITLE
Blink.cmp use with cmp sources.

### DISCUSSION THREAD

**Discussion by redoxahmii** [LazyVim/LazyVim#4707] [Q&A]
I would like to ask if blink.cmp is backwards compatible with cmp sources like that and if they are how it can be setup.
Reason i am not creating this on the blink.cmp repo is because they already have a huge back log of issues so i would like to ask the solution here and they respond very slow too.
Thank you.

**Comment by folke** [comment]
You can look into blink-compat, but I only had limited success with it. Worked for lazydev, but that has a native blink source in the meantime. didn't work for copilot for me

**Reply by redoxahmii** [reply]
Thank you for the info i tested it with copilot and cmdline and it does not seem to work for me too.
For cmdline i expected it not to but the maintainer has that in mind as upcoming features.
The differences in speed are quite a lot so i guess i can continue using at the time without using the cmdline source.
For copilot i opted to use supermaven instead which i already did not use with cmp sources and complete completions using `<tab>` so that works.
